### PR TITLE
Add new command placeholders: {channel.name}, {channel.id}, {stream.game}, {stream.title}, {my.id}, {my.name} 

### LINKED ISSUE #2943: Variable placeholders for custom commands
**Describe your issue**
Add a `{channel}` placeholder for commands that would put the current channel. This would be useful for opening moderation popout widgets, like reward queue, predictions, etc.

**Screenshots**
example

**OS and Chatterino Version**
Windows 10 - Latest Nightly

### PR DESCRIPTION
**Pull request by Mm2PL**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

~~Part of #2943. Doesn't implement alternate text so I'll keep the issue open.~~
Closes #2943.

### CODE REVIEW

**Comment by apa420** [comment]
If this closes #2943 that should also be part of the CHANGELOG entry (also shouldn't new changelog entries be at the bottom of the list for the unversioned changes with the same type.)

**Comment by ALazyMeme** [comment]
> There is also #2326

Hmm I don't remember making this susLada

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## Unversioned
 
+- Minor: Add new command placeholders (#3155)
```
Comment (pajlada) [inline CHANGELOG.md:5]: More verbose changelog entry

**Review by pajlada — CHANGES_REQUESTED** [review]
We should document the new variables on https://wiki.chatterino.com/Commands/ too

once you have a PR there we can get them merged in at the same time

make sure to include from which version onwards the new variables in https://wiki.chatterino.com/Commands/ are available

**Comment by Mm2PL** [comment]
https://github.com/Chatterino/wiki/pull/113

**Comment by ALazyMeme** [comment]
Is there potentially a use-case for filtered variables here? For example instead of `{channelid}`, we have a filter similar to pajbot `|id`. Could be used in a number of cases like: `{1|id}` which would get the user id of the user in the first argument, or `{channel|id}`, which would get the current channel's id. This would thus remove the `{channelid}` and `{myid}` variables, but opens us up for more options to use it (instead of just being limited to the variables that are already there).

pajbot filters here for ref: https://github.com/pajbot/pajbot/blob/master/docs/variables.md#available-filters

**Review by pajlada — APPROVED** [review]

File: src/widgets/helper/ChannelView.cpp [hunk 2]
```diff
@@ -2101,12 +2101,21 @@ void ChannelView::handleLinkClick(QMouseEvent *event, const Link &link,
...
+                    {"msg.id", layout->getMessage()->id},
+                    {"msg.text", layout->getMessage()->messageText},
+
+                    // old placehodlers
+                    {"user", layout->getMessage()->loginName},
+                    {"msg-id", layout->getMessage()->id},
+                    {"message", layout->getMessage()->messageText},
+
+                    // new version of this is inside execCustomCommand
+                    {"channel", this->channel()->getName()},
```
Comment (zneix) [inline src/widgets/helper/ChannelView.cpp:2117]: Will we mark these as deprecated and remove them later or are we just gonna keep these as legacy?

Comment (Mm2PL) [reply src/widgets/helper/ChannelView.cpp:2117]: I don't like breaking things, maybe we do it some time should though.

Comment (zneix) [reply src/widgets/helper/ChannelView.cpp:2117]: Yes that shouldn't be changed right now, but maybe a note for users that new variables should be used now and maybe remove this like after over a few stable releases to not keep stale legacy forever is reasonable.

**Review by pajlada — APPROVED** [review]

**Review by zneix — APPROVED** [review]

**Pull request merged**